_docs/glider-background-and-sampling-terminology.md

```diff
@@ -19,9 +19,9 @@ As of this writing, there are 3 major buoyancy driven glider types that are curr
 
 The schematic and definitions below define the sampling terminology of a profiling glider.  While all of the terms defined below are commonly used in the glider community, the 2 fundamental terms used by the **NGDAC** to organize data are the **profile** and **trajectory**.  The **NGDAC** receives glider data as individual, sequentially numbered **profiles** and aggregates files from the same **trajectory** into a single data set representing the deployment.
 
-![Glider Sampling Patterns and Terms](/glider-dac/glider-sampling-terminology.png)
+![Glider Sampling Patterns and Terms](glider-sampling-terminology.png)
```

Comment:
I removed all the hardcoded links to the `baseurl` (`/glider-dac`) so we can see them in both the rendered HTML and Markdown.